examples/ping-pong-with-noise/Cargo.toml

```diff
@@ -15,7 +15,7 @@ async-std="1.8.0"
 bytes = "1.0.1"
 binary_sv2 = { path = "../../protocols/v2/binary-sv2/binary-sv2" }
 codec_sv2 = { path = "../../protocols/v2/codec-sv2", features=["noise_sv2"] }
```

Comment:
Why there's not a version specified here?

Reply:
ack https://github.com/stratum-mining/stratum/pull/785/commits/a877dd2960832b743fdbc0bd041f89b10fb252a8